home.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * The main template file
+ *
+ * This is the most generic template file in a WordPress theme
+ * and one of the two required files for a theme (the other being style.css).
+ * It is used to display a page when nothing more specific matches a query.
+ * E.g., it puts together the home page when no home.php file exists.
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package nightingale-wp
+ */
+
+get_header(); ?>
+<div class="o-layout  o-layout--wide">
+
+	<!-- reduce width of primary content if sidebar contains widgets -->
+	<div id="primary" class="o-layout__item u-8/12@lg">
+		
+		<main id="main" class="site-main" role="main">
+
+		<?php
+		if ( have_posts() ) :
+
+			if ( ! is_front_page() ) : ?>
+				<header>
+					<h1 class="page-title screen-reader-text"><?php single_post_title(); ?></h1>
```

Comment:
What title is this outputting. If it outputs when it is not the homepage, this template will be used as a blog listing template so not sure why we need a single_post_title here?